### PR TITLE
ddl: migrate test-infra to testify for ddl/column_change_test.go

### LINKED ISSUE #28630: migrate test-infra to testify for `ddl/column_change_test.go`

### PR DESCRIPTION
**Pull request by GOODBOY008**
### What problem does this PR solve?

Issue Number: close #28630

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by tisonkun — CHANGES_REQUESTED** [review]
@GOODBOY008 please take a look at existing migration example such as under `expression` package. This patch is invalid and shows that you don't know how go testing works. Also, CI tasks failed.

You should remove "pingcap/check" dependency and migrate its `Suite` and thus `SetUpSuite`/`TearDownSuite` logics.

File: ddl/column_change_test.go [hunk 1]
```diff
@@ -45,55 +47,55 @@ type testColumnChangeSuite struct {
...
 	}()
 	// create table t (c1 int, c2 int);
 	tblInfo, err := testTableInfo(d, "t", 2)
-	c.Assert(err, IsNil)
+	require.Nil(t, err)
 	ctx := testNewContext(d)
 	err = ctx.NewTxn(context.Background())
-	c.Assert(err, IsNil)
-	testCreateTable(c, ctx, d, s.dbInfo, tblInfo)
+	require.Nil(t, err)
```
Comment (hawkingrei) [inline ddl/column_change_test.go:88]: require.NoError(t, err)

File: ddl/column_change_test.go [hunk 2]
```diff
@@ -45,55 +47,55 @@ type testColumnChangeSuite struct {
...
+	testCreateTableT(t, ctx, d, s.dbInfo, tblInfo)
 	// insert t values (1, 2);
-	originTable := testGetTable(c, d, s.dbInfo.ID, tblInfo.ID)
+	originTable := testGetTableT(t, d, s.dbInfo.ID, tblInfo.ID)
 	row := types.MakeDatums(1, 2)
 	h, err := originTable.AddRecord(ctx, row)
-	c.Assert(err, IsNil)
+	require.Nil(t, err)
```
Comment (hawkingrei) [inline ddl/column_change_test.go:94]: Ditto

**Pull request closed without merging**